apps/examples/src/e2e/TestingView.tsx

```diff
@@ -0,0 +1,154 @@
+import React, {
+  Component,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import {Platform, Text, View} from 'react-native';
+import * as RNSVG from 'react-native-svg';
+import ViewShot from 'react-native-view-shot';
+
+const address = ['ios', 'web'].includes(Platform.OS) ? 'localhost' : '10.0.2.2';
+const wsUri = `ws://${address}:7123`;
+
+const TestingView = () => {
+  const [renderedContent, setRenderedContent] = useState();
+  const [wsClient, setWsClient] = useState<WebSocket | null>(null);
+  const [readyToSnapshot, setReadyToSnapshot] = useState(false);
+  const wrapperRef = useRef<ViewShot>(null);
+  const [resolution, setResolution] = useState([600, 600]);
+  const [message, setMessage] = useState('⏳ Connecting to Jest server...');
+
+  const connect = useCallback(() => {
+    const client = new WebSocket(wsUri);
+    setWsClient(client);
+    setMessage('⏳ Connecting to Jest server...');
+    client.onopen = () => {
+      client.send(
+        JSON.stringify({
+          os: Platform.OS,
+          version: Platform.Version,
+          arch: isFabric() ? 'fabric' : 'paper',
+          connectionTime: new Date(),
+        }),
+      );
+      setMessage('✅ Connected to Jest server. Waiting for render requests.');
+    };
+    client.onerror = (err: any) => {
```

Comment:
It's just a suggestion, but have you considered using an [Error](https://stackoverflow.com/a/54649623/24106872) type instead of any?

Reply:
Typing is tricky in this part of project (whole e2e dir basically). I tried to make it as good as possible, but here it would fail for some reason. `any` is good enough for now.